core/repository/transaction_repository.hpp

```diff
@@ -18,27 +18,21 @@ See the License for the specific language governing permissions and
 #ifndef IROHA_TRANSACTION_REPOSITORY_HPP
 #define IROHA_TRANSACTION_REPOSITORY_HPP
 
+#include <stdexcept>
 #include <string>
 #include <vector>
-#include <stdexcept>
```

Comment:
Actually, I applied `clang-format` to including headers. This swapping is no meaning. Sorry...